libs/server-sdk/src/evaluation/bucketing.cpp

```diff
@@ -73,7 +73,7 @@ tl::expected<std::pair<ContextHashValue, RolloutKindLookup>, Error> Bucket(
             Error::InvalidAttributeReference(ref.RedactionName()));
     }
 
-    Value value = context.Get(context_kind, ref);
+    Value const& value = context.Get(context_kind, ref);
 
```

Comment:
This is the more worrisome one.

Reply:
We may want to consider some temporary static inspection. Track how many values are copied.